.idea/codeStyleSettings.xml

```diff
@@ -1,264 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please don't remote the files in .idea

Reply:
That also occurred in another branch. I did not consciously remove them, but I’ll keep a better eye on what files are being committed in the future.

Reply:
@thellmund !ping
please revert the changes to .idea